src/main/resources/hudson/plugins/junitattachments/TestCaseAttachmentTestAction/summary.jelly

```diff
@@ -1,23 +1,20 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
 
-	<tr><td>
-	<h3>${%Attachments}</h3>
-	<table class="jenkins-table" id="attachments">
+	<h1>${%Attachments}</h1>
```

Comment:
There shouldn't be multiple h1s on a page and heading levels should descend one-by-one.

If there's no h2 above this then it should be an h2